packages/modules/src/modules/banners/common/BannerText.tsx

```diff
@@ -24,6 +24,49 @@ type BannerTextProps = {
     children?: React.ReactNode;
 };
 
+export const createBannerBodyCopy = (
+    paras: (Array<JSX.Element> | JSX.Element)[],
+    highlights: Array<JSX.Element> | JSX.Element | null | undefined,
+    renderStyles: BannerTextStyles,
+): JSX.Element[] | JSX.Element => {
+    if (paras != null && highlights != null) {
```

Comment:
no need to check if `paras != null`, it's guaranteed to be an array at this point

Reply:
and this function can be simplified to:
```
if (highlightedText) {
    // render paras with highlightedText at end
} else {
    // render paras without highlightedText
}
```